playground/keycloak/Keycloak.AppHost/Program.cs

```diff
@@ -0,0 +1,15 @@
+var builder = DistributedApplication.CreateBuilder(args);
+
+var keycloak = builder.AddKeycloak("keycloak")
+                      .WithDataVolume()
+                      .WithRealmImport("../realms");
```

Comment:
We should add a test that uses `WithRealmImport` also.

Reply:
Added